hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -28,9 +28,17 @@ const {Sider} = Layout;
 interface INavBarProps extends RouteComponentProps<object> {
   collapsed: boolean;
   onCollapse: (arg: boolean) => void;
+  isheatmapavailable: boolean;
 }
 
 class NavBar extends React.Component<INavBarProps> {
+  constructor(props = {}) {
+    super(props);
+    this.state = {
+      isheatmapavailable: false
```

Comment:
lets keep this as default "true" and show menu and UI, For empty response, don't render empty treemap, instead show "No Data Available" or some standard message if we are following any convention. Will document this in Ozone doc.

Reply:
@devmadhuu  Completed Changes.

Reply:
I can't see that you did the changes suggested by @devmadhuu